src/transport/msgs.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(missing_docs)]
```

Comment:
Is there demo client to request on the public interface of lsp handler?  It will be nice for people to check after integrate the handler.  Many thanks.

Reply:
I forget the state it's in but I think it was working: https://github.com/johncantrell97/ldk-sample/pull/1/files